osi_hostvehicledata.proto

```diff
@@ -2,31 +2,356 @@ syntax = "proto2";
 
 option optimize_for = SPEED;
 
+import "osi_version.proto";
 import "osi_common.proto";
+import "osi_object.proto";
 
 package osi3;
 
-//
-// \brief Interface for host vehicle data that is available to sensors and
-// other functions due to host vehicle's internal communication.
+// \brief Host vehicle data is about the perception of the vehicle about it's own, internal states.
+// It describes data that the host vehicle knows about itself,
+// e.g. from location sensors, internal sensors, board net etc.
+// A dynamic model can serve as input provider.
+// Sensors, mockups or other modules can make usage of the host vehicle data.
+// It consists of different messages categorizing the vehicle in:
+// Vehicle-Basics, Vehicle-Powermanagement, Vehicle-Powertrain, Vehicle-SteeringWheel, Vehicle-Wheels, Vehicle-Localization.
 //
 // \image html OSI_HostVehicle.svg
 //
-// All coordinates and orientations are relative to the global ground truth
-// coordinate system.
+// All coordinates and orientations are relative to the global ground truth coordinate system.
+// Otherwise it is mentioned explicitly.
 //
 message HostVehicleData
 {
+    // Deprecated: Will be removed in next major release. Moved to VehiclePositionAndKinematics.
     // Current estimated location based on GPS- and related navigation sensors.
     //
     // \note Note that dimension and base_polygon need not be set.
     //
     optional BaseMoving location = 1;
 
-    // Current estimated location error based on GPS- and related navigation
+    // Deprecated: Will be removed in next major release. Moved to VehiclePositionAndKinematics.
+    // Current estimated location error based on GPS- and related navigation 
     // sensors.
     //
     // \note Note that dimension and base_polygon need not be set.
     //
     optional BaseMoving location_rmse = 2;
+
+    // The ID of the object. Can be discussed.
+    //
+    // \rules
+    // is_globally_unique
+    // \endrules
+    //
+    optional Identifier id = 3;
```

Comment:
probably should have a longer name like host_vehicle_id, to make the semantics clearer (cf. some of the identifier discussions). Or should be dropped, since it is, s you allude to, not really known to the vehicle itself, but comes from the simulation context (i.e. SensorView and TrafficUpdate already carry the host vehicle ID).

Reply:
`Optional Identified id = 3` should be referred from `osi::trafficupdate` which inturn should refer from `osi3::object.MovingObject`. Advantage: Direct linkage between `trafficupdate` and `hostvehicledata` message. In that way there is a direct linkage between `hostvehicledata` coming from a traffic participant the focused in the simulation.